ops/services/app_functions/report_stream_batched_publisher/functions/QueueBatchedReportStreamUploader/lib.ts

```diff
@@ -25,6 +25,7 @@ const {
   AZ_STORAGE_QUEUE_SVC_URL,
 } = ENV;
 const DEQUEUE_BATCH_SIZE = 32;
+const DEQUEUE_TIMEOUT = 60 * 60;  // length of time in seconds a message is invisible after being dequeued
```

Comment:
So this is changing it to an hour from 30 seconds, is that the intention?

Reply:
yes, 1 hr is a little bit arbitrary but it's a larger number that just is meant to be long enough that our function can delete the message before it reappears again.